Tests/UpstreamIntegrationTests/UpstreamIntegrationTests.swift

```diff
@@ -151,12 +142,225 @@ class UpstreamIntegrationTests: XCTestCase {
         wait(for: [edgeRequestContentExpectation, networkResponseExpectation], timeout: asyncTimeout)
     }
     
-    func test_testFailureExample() {
-        let experienceEvent = ExperienceEvent(xdm: ["xdmtest": "data"],
-                                              data: ["data": ["test": "data"]])
+    func testFlexibleValidation() {
+        let validationJSON = #"""
+        {
+          "payload": [
+            {
+              "ttlSeconds" : 1800,
+              "scope" : "EdgeNetwork",
+              "hint" : "or2"
+            }
+          ]        }
+        """#
+        
+        let inputJSON = #"""
+       {
+         "payload": [
+           {
+             "ttlSeconds" : 1800,
+             "scope" : "Target",
+             "hint" : "35"
+           },
+           {
+             "ttlSeconds" : 1800,
+             "scope" : "AAM",
+             "hint" : "9"
+           },
+           {
+             "ttlSeconds" : 1800,
+             "scope" : "EdgeNetwork",
+             "hint" : "or2"
+           }
+         ]
+       }
+       """#
+        
+        let jsonValidation = try? JSONDecoder().decode(AnyCodable.self, from: validationJSON.data(using: .utf8)!)
```

Comment:
nit: Should we update to `expectedJSON` or 'jsonExpectation`. jsonValidation sounds more like an operation rather than a data to be validated.